### ISSUE TITLE
wrong cover when opened via doubleclick in explorer

### ISSUE THREAD

**Issue by BodoTh** [digimezzo/dopamine#714]
When Dopamine is freshly opened by clicking on an mp3 file Dopamine plays the correct mp3 audio file but sometimes displays the wrong cover (even if there is a cover in the freshly opened file). This seems to happend when the previous file has not played till the end but dopamine was closed while the file was playing.

1. open an mp3 file with cover
2. shut down dopamine while the first file still is played (by closing the window)
3. double click on another mp3 file with cover

Dopamine will open and display the wrong cover.

**Comment by digimezzo** [comment]
@BodoTh Thank you for the bug report. I'll have a look at it.

**Comment by digimezzo** [comment]
@BodoTh I can't reproduce this issue. Does it happen each time? Are there other steps required to reproduce the problem?

**Comment by BodoTh** [comment]
sorry, didn't see your response.
When I tested this initially, this was 100% reproducable.
Strangely the problems got even worse (without a new version of Dopamine).
Dopamine either crashes or does other strange things:
What I did:
Dopamine was not opened.
I double clicked on an mp3 file of 12 minutes.
Playback starts.
I jump to position 8 minutes.
I close Dopamine without stopping playback (of course playback stops nonetheless).
I double click on an Opus 1.5 file with a length of 3 minutes.
Dopamine opens.
Dopamine still shows the mp3 tag information of the long file
Dopamine starts playback of the short opus file.
After 2-3 seconds Dopamine jumps back to the longer MP3 file playback.

Sadly this is not 100% reproducable. Depending on some factors (perhaps length of the MP3 file) Dopamine does skip the MP3 file completely, crashes or does as described above.
